test/parallel/test-readline-interface-escapecodetimeout.js

```diff
@@ -0,0 +1,71 @@
+'use strict';
+require('../common');
+
+// This test ensures that the escapeCodeTimeout option set correctly
+
+const assert = require('assert');
+const readline = require('readline');
+const EventEmitter = require('events').EventEmitter;
+const ESCAPE_CODE_TIMEOUT = 500;
+
+class FakeInput extends EventEmitter {
+  resume() {}
+  pause() {}
+  write() {}
+  end() {}
+}
+
+{
+  const fi = new FakeInput();
+  const rli = new readline.Interface({
+    input: fi,
+    output: fi,
+    escapeCodeTimeout: 50
+  });
+  assert.strictEqual(rli.escapeCodeTimeout, 50);
+  rli.close();
+}
+
+{
+  const fi = new FakeInput();
+  const rli = new readline.Interface({
+    input: fi,
+    output: fi,
+    escapeCodeTimeout: null
+  });
+  assert.strictEqual(rli.escapeCodeTimeout, ESCAPE_CODE_TIMEOUT);
+  rli.close();
+}
```

Comment:
These tests are doing all the same thing and are just duplicated. It would be best to write:

```js
[
  null,
  {},
  NaN,
  '50'
].forEach((invalidInput) => {
  const fi = new FakeInput();
  const rli = new readline.Interface({
    input: fi,
    output: fi,
    escapeCodeTimeout: invalidInput
  });
  assert.strictEqual(rli.escapeCodeTimeout, ESCAPE_CODE_TIMEOUT);
  rli.close();
});